hs_core/templatetags/hydroshare_tags.py

```diff
@@ -101,7 +101,7 @@ def resource_first_author(content):
             break
 
     if first_creator:
-        if first_creator.name and first_creator.relative_uri:
+        if first_creator.name and first_creator.relative_uri and first_creator.is_active_user:
```

Comment:
does this cause exception when there is no active creator on the resource? For example, create a resource, add an author, then deactivate that author's account, does the res page show error?